internal/js/summary.go

```diff
@@ -159,3 +160,10 @@ func getSummaryResult(rawResult sobek.Value) (map[string]io.Reader, error) {
 
 	return results, nil
 }
+
+func calculateCounterRate(count float64, duration time.Duration) float64 {
```

Comment:
Sounds like it should be one of the methods of the `*metrics.CounterSink`, if I'm not mistaken, we duplicate it two more times

Reply:
Yeah, good recommendation! Thanks!

Reply:
Done @ https://github.com/grafana/k6/pull/4089/commits/58e3d021ea7509bfa7aba35b0babefb7c54e04cc